src/phase_3_ransac_filtering/out.csv

```diff
@@ -0,0 +1,7086 @@
+x1,y1,sigma1,x2,y2,sigma2
```

Comment:
Is this file needed?

Reply:
out.csv is the filtered key points. I did remove it.